src/main/resources/hudson/plugins/ec2/SlaveTemplate/config.jelly

```diff
@@ -177,6 +177,10 @@ THE SOFTWARE.
     <f:entry title="${%Maximum Total Uses}" field="maxTotalUses">
       <f:textbox default="-1"/>
     </f:entry>
+    
+    	<f:entry title="Minimum uptime in hours" field="minUpHours">
+		<f:textbox default="0" />
```

Comment:
This should use `<f:number>` instead. In fact, all the other textboxes in this form that are numeric should also use that.